README.md

```diff
@@ -123,3 +123,31 @@ Volunteer Lead: Amol Kahat
 - Co-Chair: TBD
 - [Github Issues](https://github.com/pythonindia/inpycon2020-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Ftickets)
 
+## Timeline
+### TLDR;
+* Sponsership going live on 31st July, 2020.
+* Schedule shall be released on 1st September 2020.
+* Talks will be presented on 2nd and 3rd October 2020.
+* CFP closes on 14th August 2020
+* Ticket sales going live on <>.
+* Ticket sales closing date on <>.
+
+### Below is the milestones of various tasks:
+| **Workgroup** | **Owner** | **Total Target** |**Target Date Wise** | **Status** |
+|---|---|---|---|---|
+|Sponsership|Chandan Kumar||31st July, 2020|-|Going Live|
+||Sayan|||||
+||Sanchit|||||
+||Abhishek|||||
+|Reviewer|Ritesh|50|* 30th June, 2020 -> 15 | -|
```

Comment:
Let's not write the name that will be redundant information.

Reply:
Yes, removed

Reply:
The deadline will just have the dates, and Objective will have the numbers.